### PR TITLE
fixed MIG-127 and adjusted tests to include manifest in memo

### PR DESCRIPTION
**Pull request by emilyackermanmedable**
Also fixed a bug in StudyManifestTools that previously meant ec__document_templates were not being included in study exports: we had been looking for the property "c_study" in ec__document_templates, but that property does not exist; rather, it is called ec__study on that object

### CODE REVIEW

File: packages/mdctl-axon-tools/lib/StudyManifestTools.js [hunk 1]
```diff
@@ -707,7 +707,7 @@ class StudyManifestTools {
         }
         case 'ec__document_template': {
           // Get the eConsents ID's from the study or the manifest
-          ids = (await this.getObjectIDsArray(org, key, property, values)).map(v => v._id)
+          ids = (await this.getObjectIDsArray(org, key, 'ec__study', values)).map(v => v._id)
```
Comment (davideaquaro) [inline packages/mdctl-axon-tools/lib/StudyManifestTools.js:710]: Hi @emilyackermanmedable,I think that _property_ variable is used to determine if querying by 'c_study' (for full migrations) or 'c_key' (for partial migrations) when calling getObjectIDsArray method. 
If that value is hard coded , will it still work properly for partial migrations where we pass c_key instances instead?

**Review by davideaquaro — CHANGES_REQUESTED** [review]
Hi @emilyackermanmedable,I think that property variable is used to determine if querying by 'c_study' (for full migrations) or 'c_key' (for partial migrations) when calling getObjectIDsArray method.
If that value is hard coded , will it still work properly for partial migrations where we pass c_key instances instead?

File: packages/mdctl-axon-tools/lib/StudyManifestTools.js (hunk 1 above)
Comment (emilyackermanmedable) [reply packages/mdctl-axon-tools/lib/StudyManifestTools.js]: Fixed it - thank you for catching that!

**Review by davideaquaro — APPROVED** [review]
Suggested changes applied. All tests passing.

**Review by davideaquaro — APPROVED** [review]

**Review by davideaquaro — APPROVED** [review]

**Review by fiachra — APPROVED** [review]

**Pull request merged**